multistreaming-server/launch-nginx-server.sh

```diff
@@ -29,12 +29,18 @@ if [ $MULTISTREAMING_KEY_PERISCOPE ]; then
 	sed -e "s/##PUSH_PERISCOPE_MARKER##//g" -i /usr/local/nginx/conf/nginx.conf
 fi
 
-
 if [ $MULTISTREAMING_KEY_CUSTOM ]; then
 	envsubst < nginx-conf-custom.txt >>  /usr/local/nginx/conf/nginx.conf
 	sed -e "s/##PUSH_CUSTOM_MARKER##//g" -i /usr/local/nginx/conf/nginx.conf
 fi
 
+if [ $MULTISTREAMING_KEY_MICROSOFTSTREAM ]; then
+	export MICROSOFTSTREAMRTMP=${MULTISTREAMING_KEY_MICROSOFTSTREAM%/live/*}
+	export MICROSOFTSTREAMAPPNAME=live/${MULTISTREAMING_KEY_MICROSOFTSTREAM#*/live/}
+	envsubst \${MICROSOFTSTREAMRTMP},\${MICROSOFTSTREAMAPPNAME} < nginx-conf-microsoftstream.txt >>  /usr/local/nginx/conf/nginx.conf
```

Comment:
You are limiting the variable substitution to only `MICROSOFTSTREAMRTMP` and `MICROSOFTSTREAMAPPNAME`. There should be no limit because some other variables need to be substituted, most notably `DOLLAR`. Even if the `nginx-conf-microsoftstream.txt` doesn't currently need any other variable substitution, let's keep things consistent and future proof.

Reply:
I agree. Good point. It is now updated to remove those limits of the specific variables.